xs/src/libslic3r/Print.cpp

```diff
@@ -1123,6 +1130,124 @@ Print::_make_brim()
         ));
     }
     
+    if(this->config.brim_ears){
+        
+        //create ear pattern
+        coord_t size_ear = (scale_(this->config.brim_width.value) - flow.scaled_spacing());
+        Polygon point_round;
+        point_round.points.push_back(Point(size_ear*1, 0*size_ear));
```

Comment:
This can be replaced with array of x,y coefficients ...
Also maybe angle + sin/cos may be used to generate the ear?

Reply:
yes. I used that because it's quick and easy to write & test. Also we don't need a big precision and this number of point is enough. Too much and the intersection takes ages.

What code do you propose?

Reply:
something like (untested):
```
for (int i = 0; i < POLY_SIDES; i++) {
  float angle = 2 * PI / POLY_SIDES * i;
   point_round.points.push_back(Point(size_ear * cos(angle), size_ear * sin(angle));
}